### PR TITLE
BookSuggestion

### PR DESCRIPTION
**Pull request by mcolombo87**
### Summary
New model Booksuggestion.
### ScreenShoots
image
### Trello
https://trello.com/c/Owb2R50R/26-crear-modelo-de-booksuggestion

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,6 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# BookSuggestion.create(user_id:1, genre:"terror", author:"yo", image:"path",title:"TitleOne",editor:"Wolox",year:2001,synopsis:"Una Sinopsis bien hecha", price:10.2)
```
Comment (facucompe) [inline db/seeds.rb]: Remove this comment, we are not implementing a seed file

File: db/schema.rb [hunk 2]
```diff
@@ -10,18 +10,33 @@
...
+  end
+
   create_table "books", force: :cascade do |t|
-    t.string "genre"
-    t.string "author"
-    t.string "image"
-    t.string "title"
-    t.string "editor"
-    t.integer "year"
+    t.string "genre", null: false
```
Comment (facucompe) [inline db/schema.rb:34]: Please be careful with this, I should not see this changes in this PR.

**Review by facucompe — CHANGES_REQUESTED** [review]

File: db/seeds.rb (hunk 1 above)
Comment (mcolombo87) [reply db/seeds.rb]: Deleted it. That was a personal test.

File: db/schema.rb (hunk 2 above)
Comment (mcolombo87) [reply db/schema.rb:34]: When i migrated BookSuggestions rewrite schema.rb, i don't know why exactly, but this is how this file must be.

**Review by facucompe — APPROVED** [review]

**Review by floresv — APPROVED** [review]

**Pull request merged**